### PR TITLE
home page why-us-text

### PR DESCRIPTION
**Pull request by 2016DigiTech**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #why-us-text

Test URLs:
- Before: https://main--tlcgrp--hlxsites.hlx.page/
- After: https://why-us-text--tlcgrp--hlxsites.hlx.page/

### CODE REVIEW

**Comment by 2016DigiTech** [comment]
Please review

File: styles/styles.css [hunk 1]
```diff
@@ -707,9 +707,37 @@ main .columns-container {
   opacity: 0;
 }
 
-/* mobile code start here */
+main .why-us h2 {
+  padding-bottom: 20px;
+}
+
+main .why-us p{
+ font-size: 18px!important;
```
Comment (rofe) [inline styles/styles.css:715]: Using `!important` should generally be avoided as it hinders further cascading of styles. When it seemingly becomes necessary, it's usually a sign of non-ascending specificity in the selectors of rules matching for a given element. I think in this case could be avoided by changing the rule's selector to:
```css
main .section.why-us .default-content-wrapper p { ... }
```
in order to make it more specific than the previously winning rule:
```css
main .section .default-content-wrapper p { ... }
```

File: styles/styles.css [hunk 2]
```diff
@@ -707,9 +707,37 @@ main .columns-container {
...
+
+main .why-us h4 {
+  margin-top: 20px;
+  font-family: var(--alt-font-family);
+  font-weight: 200;
+}
+
+@media (max-width:600px){
+  main .why-us p{
+    font-size: 14px!important;
```
Comment (rofe) [inline styles/styles.css:735]: Same as above

File: styles/styles.css [hunk 3]
```diff
@@ -707,9 +707,37 @@ main .columns-container {
...
 }
 
-/* mobile code start here */
+main .why-us h2 {
+  padding-bottom: 20px;
+}
+
+main .why-us p{
+ font-size: 18px!important;
+ padding:8px 0px;
```
Comment (rofe) [inline styles/styles.css:716]: Linting
```suggestion
 padding:8px 0;
```

File: styles/styles.css [hunk 4]
```diff
@@ -707,9 +707,37 @@ main .columns-container {
...
+  margin-top: 20px;
+  font-family: var(--alt-font-family);
+  font-weight: 200;
+}
+
+@media (max-width:600px){
+  main .why-us p{
+    font-size: 14px!important;
+    padding:8px 0px;
+    margin:0px 3%;
```
Comment (rofe) [inline styles/styles.css:737]: Linting
```suggestion
    padding:8px 0;
    margin:0 3%;
```

**Review by rofe — CHANGES_REQUESTED** [review]

**Comment by rofe** [comment]
As a general comment, I would recommend considering a dedicated block (e.g. `why-us`) instead of using section metadata if the styles just target the default content in a given section. Section metadata are primarily meant to tweak properties of the _section_ itself: for example background color or image, font color (e.g. if you want to invert background and text for all blocks _and_ default content within a section).

**Pull request closed without merging**